uc_product/uc_product.install

```diff
@@ -230,7 +230,6 @@ function uc_product_disable() {
 function uc_product_enable() {
   $body_label = t('Description');
 
-  node_types_rebuild();
```

Comment:
Should this also have a `node_type_cache_reset()` call?

Reply:
Dunno for sure, but that makes sense and can't hurt. Added.